src/main/java/net/minecraftforge/common/util/InvocationProfiler.java

```diff
@@ -0,0 +1,37 @@
+package net.minecraftforge.common.util;
```

Comment:
I do not think we need this class - it's not a very accurate benchmarker.

Reply:
Yeah it wasn't meant to be commited. Its my better than nothing profiler